### ISSUE TITLE
100% Completed torrent does not honor "do not download"

### ISSUE THREAD

**Issue by jamesaepp** [picotorrent/picotorrent#500]
Unlike other BT clients that will move downloaded files into a .\.unwanted folder or similar, when my torrent is 100% complete, the PT client does not honor my selections. If I unselect a file and hit "OK" , it does not change the total size of the torrent (which should have decreased). Going back into the details File tab again, it has took back the file I had changed to "do not download" is now set back to its previous priority.

**Comment by jlkkljh** [comment]
have you tried deleting the file and seeing if it comes back?

**Comment by jamesaepp** [comment]
yes and no. I renamed one of the files in the torrent (it probably has over 100 files) and it did not do a re-check or anything. Not that I would expect it to. It's not like it's watching the filesystem.

**Comment by jlkkljh** [comment]
so problem is that it doesn't delete files, makes sense to me

**Comment by vktr** [comment]
Can you confirm this behaviour in the latest beta version?https://ci.appveyor.com/project/picotorrent/picotorrent/build/job/7hfeosov25j0lli7/artifacts

**Comment by Dyras** [comment]
@vktr This seems to still be the case for me. If I unclick all the rarbg.exe's and readme.nfo crap in a torrent after downloading it, it's not moved anywhere or deleted at all.

**Comment by jlkkljh** [comment]
What a surprise to see this thread in my inbox

**Comment by jlkkljh** [comment]
Try setting it before it downloads the file, if that's the mistake your making, sorry to say, but thought I would rule out user error. 

I switched to flud on android or fdm5 on Windows. Hate to be"that guy" 
good luck main dev. I'd start with the queueing system and change it so that it compares the file blacklist with queue first thing forevery file. 
Queue<file name<is blacklist<start data stream

**Comment by Dyras** [comment]
@jlkkljh I'm aware this works if you unclick stuff before downloading. But if you unclick the files **after** it has downloaded, the behavior is still different from what's expected coming from more or less any other client.

**Comment by jlkkljh** [comment]
To be honest? That's The kind of thing to look for before downloading. I expect nothing more. Imo it's the users fault for changing their mind. It's like eating a while pie and changing your mind after " I'll just put this back"

**Comment by vktr** [comment]
Hello 👋 Sorry your having issues! @Dyras you mention the behavior differs from other clients - I don't have the possibility to test right now, but what is the expected behavior if you select "Do not download" for a file that is downloaded?

**Comment by jlkkljh** [comment]
@vktr I'm glad you caught that too. 

I think the threat name should be changed to feature request

**Comment by Dyras** [comment]
@vktr In qBittorrent and uTorrent, they files are put in a separate, hidden folder.

**Comment by vktr** [comment]
Thanks @Dyras! That seems like a reasonable feature
